qiskit_ibm_runtime/accounts/management.py

```diff
@@ -61,6 +62,7 @@ def save(
             proxies=proxies,
             verify=verify,
             channel_strategy=channel_strategy,
+            private_endpoint=private_endpoint if not url else False,
```

Comment:
Ignoring `private_endpoint` when `url` param is provided only makes sense if the `url` is kept unchanged. If we change the `url` with the location, then we have to always take into account the `private_endpoint` param. So the condition is not necessary.